cognite/client/config.py

```diff
@@ -82,13 +82,16 @@ def apply_settings(self, settings: dict[str, Any] | str) -> None:
 
         loaded = load_resource_to_dict(settings)
 
+        if not set(loaded).issubset(set(self.__dict__)):
```

Comment:
We can avoid converting both of the key views to sets:
```suggestion
        if not loaded.keys() <= self.__dict__.keys():
```

Reply:
Lol, I guess you can save a `not` by flipping the crocodile 😅